stats/release_queue.php

```diff
@@ -291,75 +234,35 @@ function _show_queue_details($round, $name, $unheld_only)
     }
     echo "</p>";
 
-
-    $comments_url1 = DPDatabase::escape("<a href='$code_url/project.php?id=");
-    $comments_url2 = DPDatabase::escape("'>");
-    $comments_url3 = DPDatabase::escape("</a>");
-
-    $unheld_only_sql = $unheld_only ? "project_holds.state is NULL" : "1";
-    dpsql_dump_themed_query("
-        SELECT
-
-            concat('$comments_url1',projectID,'$comments_url2', nameofwork, '$comments_url3') as '"
-                . DPDatabase::escape(_("Title")) . "',
-            authorsname as '" . DPDatabase::escape(_("Author")) . "',
-            language    as '" . DPDatabase::escape(_("Language")) . "',
-            genre       as '" . DPDatabase::escape(_("Genre")) . "',
-            difficulty  as '" . DPDatabase::escape(_("Difficulty")) . "',
-            username    as '" . DPDatabase::escape(_("Project Manager")) . "',
-            FROM_UNIXTIME(modifieddate) as '"
-                . DPDatabase::escape(_("Date Last Modified")) . "',
-            IF(ISNULL(project_holds.state),'&nbsp;','Y') AS '" . DPDatabase::escape(_("Hold?")) . "'
-        FROM projects
-            LEFT OUTER JOIN project_holds USING (projectid, state)
-        WHERE ($cooked_project_selector)
-            AND state='{$round->project_waiting_state}'
-            AND $unheld_only_sql
-        ORDER BY modifieddate, nameofwork
-    ");
-}
-
-function _get_num_projects_and_pages_released_in_last($name, $seconds_ago, $waiting_state)
-{
-    $sql = sprintf(
-        "
-        SELECT count(*)
-        FROM project_events
-        WHERE event_type = 'transition'
-            AND details1 = '%s'
-            AND details3 = 'via_q: %s'
-            AND timestamp >= %d
-        ",
-        DPDatabase::escape($waiting_state),
-        DPDatabase::escape($name),
-        $seconds_ago
-    );
-    $result = DPDatabase::query($sql);
-    $projects_released = mysqli_fetch_row($result)[0];
-
-    $sql = sprintf(
-        "
-        SELECT sum(n_pages)
-        FROM projects
-        WHERE projectid in (
-            SELECT projectid
-            FROM project_events
-            WHERE event_type = 'transition'
-                AND details1 = '%s'
-                AND details3 = 'via_q: %s'
-                AND timestamp >= %d
-        )
-        ",
-        DPDatabase::escape($waiting_state),
-        DPDatabase::escape($name),
-        $seconds_ago
-    );
-    $result = DPDatabase::query($sql);
-    $pages_released = mysqli_fetch_row($result)[0] ?? 0;
-
-    return [$projects_released, $pages_released];
+    // TODO(bpfoley) Add a dump_themed_array(), perhaps?
```

Comment:
Totally grok the desire but this gets back into `dpsql_dump_themed_query_result()`-esque which is not a road I want to go down because turns out we have very few standard tables to output. I have to believe there's a better, more frameworky way to handle this.